test/utils/testutils.fypp

```diff
@@ -123,7 +122,7 @@ contains
                   do i = 1, m
                      if (.not. close (x(i, j), y(i, j), rtol_l, atol_l)) then
                         aclose = .false.
-                        return
+                        write (error_unit, *) i, j, x(i, j), y(i, j)
                      end if
                   end do
                end do
```

Comment:
Could you explain these changes?

Reply:
See https://github.com/eth-cscs/DLA-Future-Fortran/pull/40#issuecomment-2935319749.